osbs/api.py

```diff
@@ -166,7 +185,19 @@ def _create_build_config_and_build(self, build_json, namespace):
             # if it doesn't exist, then create it
             logger.debug('build config for %s doesn\'t exist, creating...', build_config_name)
             self.os.create_build_config(json.dumps(build_json), namespace=namespace)
-        return self.os.start_build(build_config_name, namespace=namespace)
+            # under some circumstances, creating a buildConfig might instantiate it, so
+            #  check for an already running build and return it if it exists
+            #  see https://github.com/projectatomic/osbs-client/issues/205
```

Comment:
Be it atomic or not, I guess it'll be safer to assume that it's not in the long run... I'll create another commit that fixes this.